OmsAgent/HandlerManifest.json

```diff
@@ -3,11 +3,11 @@
     "name":  "OmsAgentForLinux",
     "version": "1.12.25",
     "handlerManifest": {
-      "installCommand": "omsagent.py -install",
-      "uninstallCommand": "omsagent.py -uninstall",
-      "updateCommand": "omsagent.py -update",
-      "enableCommand": "omsagent.py -enable",
-      "disableCommand": "omsagent.py -disable",
+      "installCommand": "shim.sh -install",
```

Comment:
so I thought about that ... I felt that if you have `omsagent.py` and `omsagent.sh` it may seem initially confusing where the entrypoint is (without looking at this file), plus then their names are almost the same ... `shim.sh` more clearly delineates the purpose of this file. 

A compromise could be `omsagent_shim.sh` ?

Reply:
yes, good one. For example, if a customer sends us ps aux output; we can tell better...